### PR TITLE
Added flag to add new lab and new user to billing system.

### PR DESCRIPTION
**Pull request by andriy-safe-ai**
Tested script on dev cluster.

### CODE REVIEW

File: bin/onboard.sh [hunk 1]
```diff
@@ -27,10 +35,20 @@ else
     cluster group create "$lab_name" || error_exit "Failed to create lab group: $lab_name"
     # Add slurm lab account
     sudo sacctmgr add account --immediate "$lab_name" Parent="$parent_account" Description="$lab_name Lab" Organization=Prof_"$lab_name" || error_exit "Failed to add Slurm lab account for: $lab_name"
+
+    if [ "$add_billing" = true ]; then
+        group_id=$(cluster group list | grep -wA1 "$lab_name" | grep gidNumber | awk '{print $2}')
+        mysql --defaults-extra-file=/home/ubuntu/.billing.cnf -e "INSERT INTO billing.accounts (account_id, account_name, email, billing_address) VALUES ($group_id, '$lab_name', '$billing_email', '$billing_address');"
+        # Add pricing information
```
Comment (steven-basart) [inline bin/onboard.sh:42]: Odd that only this line is commented

**Review by steven-basart — CHANGES_REQUESTED** [review]
Remove the comment I think or add more if you think they're providing value.

**Review by xksteven — APPROVED** [review]

**Review by steven-basart — APPROVED** [review]
LGTM

**Comment by andriy-safe-ai** [comment]
Rebased to one commit.

**Pull request merged**